### ISSUE TITLE
Service center search button

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1899]
> The search button under "Your Local Services" in My Services tab panel does not have an accessible name (i.e., it's a blank button for screen reader users). It also doesn't seem to function for any type of user. 
`<button class="btn btn-lg btn-default hidden-xs"><i aria-hidden="true" class="fa fa-search"></i></button>`
> Recommend adding a hidden name of "Search" and making it a fully functional button like the site search button.
